packages/frontend/src/components/List/Row.tsx

```diff
@@ -281,6 +281,13 @@ export class Row extends React.Component<Row.Props, Row.State> {
       setting: 'blocklasttime',
       render: ({ blockTimestamp }) => <Ago when={blockTimestamp} />
     },
+    {
+      label: 'NetworkState',
+      icon: networkIdIcon,
+      width: 100,
+      setting: 'networkstate',
+      render: ({ id }) => React.createElement('a', {href: `/network_state/0/${id}/`, target: "_blank"}, "Network state"),
```

Comment:
What would be the best way to get the `chainLabel` here?

Reply:
You could use `getHashData().chain` (from utils), which will pull it from the url. You might want to url encode it (and decode it on the backend).